### DISCUSSION TITLE
Is the lyrics function provided by 10.9 supported in this andriod client yet?

### DISCUSSION THREAD

**Discussion by Reapor-Yurnero** [jellyfin/jellyfin-android#1442] [Q&A] [answered]
I looked up all the pr and commits and didn't see anything relevant. Am I missing anything? If not, is there an expected timeline to implement that? Thank you!

**Comment by nielsvanvelzen** [comment, marked as answer]
The Android app wraps the web client for the most part. As such, it already works!

**Reply by Reapor-Yurnero** [reply]
oops it seems i installed the 10.8 version rather than the latest. Let me try again.

**Reply by Reapor-Yurnero** [reply]
Okay it seems the latest version does have a lyrics support. But it seems it doesn't support synced lyrics at this point. Is that true or it's me?

**Reply by nielsvanvelzen** [reply]
Correct, the web client does not scroll automatically.

**Comment by mattague** [comment]
I've been playing around with my music in jellyfin, using a couple third party music apps that support synced lyrics, and I wanted to upload some I found from my phone. The android client only recognizes `example.txt` files, not `example.lrc` files, as eligible to upload. The web client does not have this limitation. The lyrics have to be in lrc format to work in most situations. As I usually notice this when not home, the easiest way to fix them is downloading them on my phone and uploading, but I noticed I had to go into a file manager, chenge the extension, upload, then ssh into the file server and change it back, in order to use them.
